test/e2e/autoscaling_utils.go

```diff
@@ -103,6 +103,7 @@ func (rc *ResourceConsumer) sendConsumeCPUrequests(requests, millicores, duratio
 
 // sendOneConsumeCPUrequest sends POST request for cpu consumption
 func (rc *ResourceConsumer) sendOneConsumeCPUrequest(millicores int, durationSec int) {
+	defer GinkgoRecover()
```

Comment:
when err (from sending post request) is not nil, then we need defer GinkgoRecover().
sendOneConsumeCPUrequest is always in a new gorutine.